editor/js/Sidebar.Scene.js

```diff
@@ -522,13 +522,13 @@ function SidebarScene( editor ) {
 
 	signals.scriptAdded.add( function () {
 
-		signals.objectChanged.dispatch( editor.selected );
+		if ( editor.selected !== null ) signals.objectChanged.dispatch( editor.selected );
 
 	} );
 
 	signals.scriptRemoved.add( function () {
 
-		signals.objectChanged.dispatch( editor.selected );
+		if ( editor.selected !== null ) signals.objectChanged.dispatch( editor.selected );
```

Comment:
The assumptions made in #28270 are not right. When scripts are added or removed, an object is _not_ necessarily selected. So when undo/redo script commands, `editor.selected` can be `null`. In this case, a runtime error occurs in `Viewport.Controls.js`.